lib/ansible/modules/packaging/os/zypper_repository.py

```diff
@@ -107,6 +114,13 @@
     repo: 'ftp://download.nvidia.com/opensuse/12.2'
     state: absent
 
+# Add NVIDIA repository for graphics drivers with additional arguments
+- zypper_repository:
+    name: nvidia-repo
+    repo: 'ftp://download.nvidia.com/opensuse/12.2'
```

Comment:
Please don't use a 12.2 repo for the example. I'm aware this is completely unrelated to your changes, but we really shouldn't use examples with versions that are EOL for several years already.

You can also update/refactor the other examples if you like. Otherwise I might clean them up later.
```suggestion
    repo: https://download.nvidia.com/opensuse/leap/15.0
```

Reply:
Thanks for the feedback